components/nav/NavUser.vue

```diff
@@ -19,16 +19,32 @@ const { busy, oauth, singleInstanceServer } = useSignIn()
     </template>
   </VDropdown>
   <template v-else>
-    <button
+    <div
       v-if="singleInstanceServer"
       flex="~ row"
-      gap-x-1 items-center justify-center btn-solid text-sm px-2 py-1 xl:hidden
-      :disabled="busy"
-      @click="oauth()"
+      gap-x-1 items-center justify-center pl-16px xl:pl-28px sm:pr-2 xl:hidden
     >
-      {{ $t('action.sign_in') }}
-    </button>
-    <button v-else btn-solid text-sm px-2 py-1 text-center xl:hidden @click="openSigninDialog()">
+      <button
+        flex="~ row"
+        gap-x-1 items-center justify-center text-sm p-x-10px p-y-10px xl:hidden text-center font-600
+        sm:mr-8px sm:btn-solid sm:btn-outline
+        :disabled="busy"
+        @click="oauth('signup')"
+      >
+        {{ $t('action.create_account') }}
```

Comment:
Re-used the translation entry for the desktop button

Reply:
Can we add the `sm:b-rd-8px` to the Create Account button as well? That way the two buttons will have a consistent border radius on all screen sizes.

<img width="295" alt="Screenshot 2023-09-08 at 5 02 52 PM" src="https://github.com/MozillaSocial/elk/assets/2893463/e342102a-35ad-4e54-ae20-0fe0566f4719">

Reply:
+1 !

Reply:
Yup! Pushed up one-line change